webapp/channels/src/components/emoji_picker/components/emoji_picker_category.tsx

```diff
@@ -39,19 +41,16 @@ function EmojiPickerCategory({category, categoryRowIndex, selected, enable, onCl
             id={`emojiPickerCategoryTooltip-${category.name}`}
             placement='bottom'
             title={
-                <FormattedMessage
-                    id={`emoji_picker.${category.name}`}
-                    defaultMessage={category.message}
-                />
+                <FormattedMessage {...category.label}/>
             }
         >
             <a
                 className={className}
                 href='#'
                 onClick={handleClick}
-                aria-label={category.id}
+                aria-label={intl.formatMessage(category.label)}
```

Comment:
This is the bug fix for the aria-label being values like `emoji_picker.recent` instead of `Recent`